org.eclipse.wildwebdeveloper.xml/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: XML support using Language Server
 Bundle-SymbolicName: org.eclipse.wildwebdeveloper.xml;singleton:=true
-Bundle-Version: 0.1.0.qualifier
+Bundle-Version: 0.1.1.qualifier
```

Comment:
The XML bundle wasn't modified, so no need to produce a new version so far. Let's keep current 0.1.0 until we modify something in the bundle.

Reply:
As it's not changed, this needs to stay to 0.1.0